static/src/javascripts/projects/commercial/modules/third-party-tags/imr-worldwide.ts

```diff
@@ -58,8 +58,10 @@ const guMetadata = {
 	'brand-only': 'P0EE0F4F4-8D7C-4082-A2A4-82C84728DC59',
 };
 
-const onLoad = () => {
-	const sectionFromMeta = config.get('page.section', '').toLowerCase();
+const onLoad = (): void => {
+	const sectionFromMeta = config
+		.get<string>('page.section', '')
```

Comment:
would using the window.guardian.page object be better? Using the types specified there?

Reply:
Good idea!